app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -161,10 +161,10 @@
               <div class="six wide column">
                 <label class="required">{{t 'Ticket Name'}}</label>
               </div>
-              <div class="column">
+              <div class="two wide column">
```

Comment:
@mrsaicharan1 If this how the new alignment should look like (your recent ss) , then you can discard the table structure I provided. Try that and send a screenshot, IMO, it would come out with a better look

Reply:
Will try it out.

Reply:
@uds5501 I made some additional changes on top of that. But it's fine, they look similar